compendium/ConfigurationAdmin/src/ConfigurationAdminImpl.cpp

```diff
@@ -394,16 +395,21 @@ std::vector<ConfigurationAddedInfo> ConfigurationAdminImpl::AddConfigurations(
       if (it == std::end(configurations)) {
         auto factoryPid = getFactoryPid(pid);
         AddFactoryInstanceIfRequired(pid, factoryPid);
+        auto newConfig = std::make_shared<ConfigurationImpl>(
+                           this,
+                           pid,
+                           std::move(factoryPid),
+                           std::move(configMetadata.properties));
+        // GetChangeCount can throw an exception if the configuration has
+        // been removed from the repository. No exception is possible here
+        // because the configuration object has not yet been added to the repository.
+        changeCount = newConfig->GetChangeCount();
```

Comment:
I'm not sure I understand the "why" of this comment. There is no way that `GetChangeCount()` can throw here. I don't understand why it's mentioned.